### PR TITLE
Documentation filter reports

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1296

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 5 sec

**Comment by dimasciput** [comment]
@Jeremy-Prior please review

File: docs/src/user/manual/explore/reports.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ The main area of the page displays your dataset in a tabular format. Each row re
 * **Activity Report:** Report on activity basis.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/reports.md:38]:
```suggestion

* **Activity Report:** Report on activity basis.
```

File: docs/src/user/manual/explore/reports.md [hunk 2]
```diff
@@ -50,9 +50,184 @@ User group is configurable from the User Table in the [Django Admin](../../../ad
 
 Activity Report columns consisted of base columns and custom columns.
 1. **Base columns**: Hardcoded.
-2. **Custom columns**: Configurable from the Django admin site by the site administrator. 
+2. **Custom columns**: Configurable from the Django admin site by the site administrator.
+
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/reports.md:54]:
```suggestion
Activity Report columns consisted of base columns and custom columns.

1. **Base columns**: Hardcoded.
2. **Custom columns**: Configurable from the Django admin site by the site administrator.
```

File: docs/src/user/manual/explore/reports.md [hunk 3]
```diff
@@ -50,9 +50,184 @@ User group is configurable from the User Table in the [Django Admin](../../../ad
...
+     - **Property Size (Ha)**: The size of the property in hectares.
+
+     - **Area Available To Species**: The portion of the property that is available for species habitat.
+
+     - **Open/Close Systems**: Indicates whether the property employs open or closed systems for wildlife management.
+
+     **Summary**
+
+     The `Property Report` is a critical filter for reporting on various properties. It provides information on property details, ownership, type, location, and specific data related to species habitat and wildlife management systems. This report is essential for analyzing and managing property-related information in an organized and structured manner.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/reports.md:157]:
```suggestion
     The `Property Report` is a critical filter for reporting on various properties. It provides information on property details, ownership, type, location, and specific data related to species habitat and wildlife management systems. This report is essential for analysing and managing property-related information in an organized and structured manner.
```

File: docs/src/user/manual/explore/reports.md [hunk 4]
```diff
@@ -50,9 +50,184 @@ User group is configurable from the User Table in the [Django Admin](../../../ad
...
+
+     - **Property Name**: The name of the property where the activity took place.
+     - **Scientific Name**: The scientific name of the species involved in the activity.
+     - **Common Name**: The common name of the species.
+     - **Year**: The year in which the activity occurred.
+     - **Total**: The total count of the species involved in the activity.
+     - **Adult Male**: The count of adult male individuals.
+     - **Adult Female**: The count of adult female individuals.
+     - **Juvenile Male**: The count of juvenile male individuals.
+     - **Juvenile Female**: The count of juvenile female individuals.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/reports.md:117]: Please use `*` for unordered lists to avoid linting errors

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

File: docs/src/user/manual/explore/reports.md (hunk 4 above)
Comment (ketankartoza) [reply docs/src/user/manual/explore/reports.md:117]: okay

Comment (ketankartoza) [reply docs/src/user/manual/explore/reports.md:117]: okay

**Comment by ketankartoza** [comment]
@Jeremy-Prior Please check

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 6 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 3 min 14 sec

**Pull request merged**